### PR TITLE
implemented declaration page - acquirer

### PR DESCRIPTION
**Pull request by AShaww**
Implemented the declaration page in the acquirer journey

### CODE REVIEW

**Review by kjoshi — CHANGES_REQUESTED** [review]
Thanks Amir, mostly looks good to me!

One thing to add:
Once the `Send data share request` button is pressed we also need to update the status of the form to `AWAITING REVIEW`.

Maybe the easiest thing would be to just add another check to the `POST` route. Something like:
```
if(req.body.continueButton && formStep === 'declaration') {
    formdata.status = "AWAITING REVIEW"
}
```

It might also be nice if you could update the type of the `FormData.status` property in `express.d.ts`. Maybe something like:

```
interface FormData {
  ...
  status: FormStatus;
  ...
}

type FormStatus = "IN PROGRESS" | "AWAITING REVIEW" ......etc.
```
(plus the other types shown in [DMBM-733](https://cddodatamarketplace.atlassian.net/jira/software/projects/DMBM/boards/6?selectedIssue=DMBM-733))

File: src/types/express.d.ts [hunk 1]
```diff
@@ -148,6 +148,15 @@ type LawfulBasisSpecialPublicInterestStep = {
 
 type MoreOrganisationStep = string[];
 
+type FormStatus =
+  | "NOT STARTED"
+  | "IN PROGRESS"
+  | "CANNOT START YET"
+  | "NOT REQUIRED"
```
Comment (kjoshi) [inline src/types/express.d.ts:155]: These two are only relevant to particular steps on the form, not the form as a whole. So I think we can get rid of them.
```suggestion
```

**Review by kjoshi — CHANGES_REQUESTED** [review]

**Review by kjoshi — APPROVED** [review]
Looks good thanks Amir 👍

**Review by madz-lw — APPROVED** [review]
LGTM 🙂

**Pull request merged**